src/proofs/Term.h

```diff
@@ -16,10 +16,8 @@ class Term {
     enum terminalType { VAR, REAL, INT, SORT, BOOL, UNDECLARED };
     virtual termType getTermType() = 0;
     virtual terminalType getTerminalType() = 0;
-    virtual void accept(class PrintVisitor *) = 0;
+    virtual void accept(class VoidVisitor *) = 0;
```

Comment:
I don't think `class` is needed here.
```suggestion
    virtual void accept(VoidVisitor *) = 0;
```

Reply:
You need to keep it as the class is only defined later.